login-workflow/src/screens/InviteRegistrationPager.tsx

```diff
@@ -393,15 +397,27 @@ export const InviteRegistrationPager: React.FC = () => {
     return (
         <BrandedCardContainer loading={registrationIsInTransit || isValidationInTransit}>
             {errorDialog}
-            <CardHeader
-                title={
-                    <Typography variant={'h6'} style={{ fontWeight: 600 }}>
-                        {pageTitle()}
-                    </Typography>
-                }
-                className={sharedClasses.dialogTitle}
-            />
-            {!accountAlreadyExists && validationSuccess && !isValidationInTransit ? (
+            {customSuccess && !accountAlreadyExists && isLastStep && (
+                <>
+                    {typeof customSuccess === 'function' &&
+                        customSuccess({ accountDetails: accountDetails, email: validationEmail })}
+                    {typeof customSuccess !== 'function' && customSuccess}
```

Comment:
Are you planning to swap this for the conditional return statements to simplify the logic?